docs/development/extensions-contrib/ddsketch-quantiles.md

```diff
@@ -0,0 +1,136 @@
+
+The `ddSketch` aggregator operates over raw data and precomputed sketches.
+
+```json
+{
+  "type" : "ddSketch",
+  "name" : <output_name>,
+  "fieldName" : <input_name>,
+  "relativeError" : <double(0, 1)>,
+  "numBins": <int>
+ }
+```
+
+|property|description|required?|
+|--------|-----------|---------|
+|type|Must be "ddSketch" |yes|
+|name|A String for the output (result) name of the calculation.|yes|
+|fieldName|A String for the name of the input field (can contain sketches or raw numeric values).|yes|
+|relativeError|Describes the precision in which to store the sketch. Must be a number between 0 and 1.|no, defaults to 0.01 (1% error)|
+|numBins|Total number of bins the sketch is allowed to use to describe the distribution. This has a direct impact on max memory used. The more total bins available, the larger the range of accurate quantiles. With relative accuracy of 2%, only 275 bins are required to cover values between 1 millisecond and 1 minute. 800 bins are required to cover values between 1 nanosecond and 1 day.|no, defaults to 1000|
+
+
+### Post Aggregators
+
+Users can query for a set of quantiles using the `quantilesFromDDSketch` post-aggregator on the sketches created by the `ddSketch` aggregators.
+
+#### quantilesFromDDSketch
+
+|property|description|required?|
+|--------|-----------|---------|
+|type|Must be "quantilesFromDDSketch" |yes|
+|name|A String for the output (result) name of the calculation.|yes|
+|field|A computed ddSketch.|yes|
+|fractions|list of doubles from 0 to 1 of the quantiles to compute|yes|
+
+#### quantileFromDDSketch
+
+|property|description|required?|
+|--------|-----------|---------|
+|type|Must be "quantileFromDDSketch" |yes|
+|name|A String for the output (result) name of the calculation.|yes|
+|field|A computed ddSketch.|yes|
+|fraction|double from 0 to 1 of the quantile to compute|yes|
+
+
+
+```json
+{
+  "type"  : "quantilesFromDDSketch",
+  "name" : <output_name>,
+  "field" : <reference to DDSketch>,
+  "fractions" : <array of doubles in [0,1]>
+}
+```
+
+Single quantiles may be fetched as well.
+
+```json
+{
+  "type"  : "quantileFromDDSketch",
+  "name" : <output_name>,
+  "field" : <reference to DDsketch>,
+  "fraction" : <double [0,1]>
+}
+```
+
+### Example
+As an example of a query with sketches pre-aggregated at ingestion time, one could set up the following aggregator at ingest:
+
+```json
+{
+  "type": "ddSketch",
+  "name": "sketch",
+  "fieldName": "value",
+  "relativeError": 0.01,
+  "numBins": 1000,
+}
+```
+
+and make queries using the following aggregator + post-aggregator:
```

Comment:
```suggestion
You can query pre-aggregated sketches using the following aggregator and post-aggregator:
```
I'm not sure if this changes the meaning of the sentence; please confirm

Reply:
What person perspective should I be writing these docs? I feel like the sudden introduction of `You` here and in the postAggregator section feels jarring to me.

I have altered the suggested changes to:
```
To compute approximate quantiles, use `quantilesFromDDSketch` to query for a set of quantiles or `quantileFromDDSketch` to query for a single quantile. Call these post-aggregators on the sketches created by the `ddSketch` aggregators.        
```
and

```
Quantiles can queried out from the pre-aggregated sketches using the following aggregator + post-aggregator:
```

Reply:
Try to use second person in the docs -- "you" instead of "we." See the docs style guide here: https://github.com/apache/druid/blob/master/docs/development/docs-contribute.md#style-checklist

The first change looks good. For the second one, "can queried out" seems odd, but perhaps this instead:
```
Compute quantiles from the pre-aggregated sketches using the following aggregator and post-aggregator:
```